### PR TITLE
update security docs

### PR DESCRIPTION
**Pull request by brad-Simeon**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -10,13 +10,11 @@ Access to the Web Admin is protected via Azure AD authentication using OIDC with
 
 ### Authenticating with the tenant
 
-Simeon can use service principal authentication, delegated authentication and/or a service account to interact with your tenants.
-
-If you use service principal authentication, you must still use either delegated authentication or service account authentication in conjuction in order for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
+ By default, Simeon uses a service principal to authenticate into in the tenant and manage supported configurations. A user account, either delegated authentication or a service account, is still required for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
```
Comment (jeff-simeon) [inline security.md:13]: A user account, authenticating with device code or password authentication, is still required....

Comment (brad-Simeon) [reply security.md]: Made the change

**Review by jeff-simeon — DISMISSED** [review]

File: security.md [hunk 2]
```diff
@@ -10,13 +10,11 @@ Access to the Web Admin is protected via Azure AD authentication using OIDC with
 
 ### Authenticating with the tenant
 
-Simeon can use service principal authentication, delegated authentication and/or a service account to interact with your tenants.
-
-If you use service principal authentication, you must still use either delegated authentication or service account authentication in conjuction in order for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
+ By default, Simeon uses a service principal to authenticate into in the tenant and manage supported configurations. A user account, authenticating with device code or password authentication, is still required for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
```
Comment (ashek-simeon) [inline security.md:13]:
```suggestion
By default, Simeon uses a service principal to authenticate into in the tenant and manage supported configurations. A user account, authenticating with device code or password authentication, is still required for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
```

File: security.md [hunk 3]
```diff
@@ -25,12 +23,12 @@ When using delegated authentication or service account authentication, Simeon us
 
 ### Approvals
 
-Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made.
+Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made. Specific users or groups can be granted access to approve changes to ensure control over this process.
```
Comment (ashek-simeon) [inline security.md:26]: I am wondering if we can get the same value out of this sentence with fewer words. No worries if you prefer not to take the below change

```suggestion
Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made. Specific users or groups can be granted access to approve changes.
```

**Review by ashek-simeon — DISMISSED** [review]

**Review by david-simeon — DISMISSED** [review]

**Review by jeff-simeon — APPROVED** [review]

File: security.md [hunk 4]
```diff
@@ -25,12 +23,12 @@ When using delegated authentication or service account authentication, Simeon us
...
 
-### Support 
+### Support
 
 Simeon support can be granted read-only or contributor access to your Azure DevOps environment to help troubleshoot any issues. Simeon support will never make changes to your tenants and can be explicitly denied permissioned to do so.
 
 ### Code Signing
 
-We digitally sign our software and the processes that run in Azure DevOps confirm the integrity of the software before interacting with your tenant. 
+We digitally sign our software and the processes that run in Azure DevOps confirm the integrity of the software before interacting with your tenant.
```
Comment (ashek-simeon) [inline security.md:34]: is this sentence missing a word somewhere?

**Review by ashek-simeon — DISMISSED** [review]

File: security.md (hunk 4 above)
Comment (ashek-simeon) [inline security.md:34]:
```suggestion
We digitally sign our software and the processes that run in Azure DevOps to confirm the integrity of the software before interacting with your tenant.
```

Comment (ashek-simeon) [reply security.md:34]: my mistake, I get it now, thank you!

**Review by ashek-simeon — APPROVED** [review]

**Review by Lance-Simeon — APPROVED** [review]

**Review by rdorta-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]

**Pull request merged**